packages/drivers/odsp-driver/src/getUrlAndHeadersAndOptionsWithAuth.ts

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
Copyright header please.

Reply:
I think we dont need this file.

Reply:
I removed this file in my latest changes, trying to fix one error( error  Expected property shorthand  object-shorthand) when I change onsCall: isOptionsCall ? true : false, to  isOptionsCall: isOptionsCall. Will upload new changes as soon as I figure out a way to address this error. Adding eslint flags didn't help so far.

Reply:
just use  isOptionsCall, when the property name and the var you are assigning is same then just doing,
{
x,
} if fine. You don't need to do { x:x, }